.github/renovate.json5

```diff
@@ -0,0 +1,183 @@
+{
```

Comment:
We are introducting another renovate file which may conflict with the original one.
This was possible because optimize uses json5 for the renovate file.
I think we need to delete our renovate file completely.

Reply:
Agreed, this makes sense. We need to adopt the shared renovate file. If necessary, we can make custom modifications to it as needed - however, the shared config file might already be mostly sufficient for our needs